engines/director/util.cpp

```diff
@@ -1224,13 +1224,13 @@ Common::String formatStringForDump(const Common::String &str) {
 // Director 2.0-4.0 Mac, MacRoman encoding
 //
 // ................................
-//   !"«»“”#$%&'‘’()*+,-./012345678
-// 9:;<=>?@AÁÀÂÄÃÅaáàâäãåÆæBbCÇcçDd
-// EÉÈÊËeéèêëFfﬁﬂGgHhIÍÌÎÏı.íìîïJjK
-// kLlMmNÑnñOÓÒÔÖΩØoóòôöõøŒœPpQqRrS
-// sßTtUÚÙÛÜuúùûüVvWwXxYyÿŸZz[\]^_`
-// {|}~.†°¢£§•¶®©™´¨≠∞±≤≥¥µ∂∑∏π∫ªºΩ
-// ¿¡¬√ƒ≈Δ…–—÷◊⁄¤‹›‡·‚„‰��¯˘˙˚¸˝˛.
+//  "#()*+,/0123456789:;<=>?@ABCDEF
```

Comment:
It is nit clear to me why are these chaged

Reply:
Fixed